src/app/app.component.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have the 'fake-commerce' title`, () => {
+  it('should have the "fake-commerce" title', () => {
     const fixture = TestBed.createComponent(AppComponent);
```

Comment:
Since each test case will use the fixture you can create a variable in the beforeEach block to handle the repetition